Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -485,16 +488,24 @@ def get_symbol_list(exchanges):
     return list(set(result))
 
 
+def _is_legit_currency(currency):
+    return not any(sub_name in currency for sub_name in EXCLUDED_CURRENCY_SUBNAME) and len(currency) < 30
```

Comment:
Why did you need `len(currency) < 30` ? Couldn't we have a legit currency with len(currency_name) > 30 ?

Reply:
From what I've seen, these currencies are "RealToken 18900 Mansfield St Detroit MI", "0-5x-long-ethereum-classic-token" etc. Do you have an example of a legit >30 char currency name ?

Reply:
No, I was suprised about that but it seems to be a legit maximum